NewEvaluation.html

```diff
@@ -23,56 +23,62 @@
 
 
 	<div class = "mainBlock">
-
+	<div class="spacing"></div>
 		<form >
-			<input type="text" name="Candidate" autocomplete="on" placeholder="Candidate" required />
-			<input type="text" name="Interviewer" autocomplete="on" placeholder="Interviewer" required />
-			<input type="date" name="Date" placeholder="dd/mm/yyyy" required/>
+			<input class="input-candidate" type="text" name="Candidate" autocomplete="on" placeholder="Candidate" required />
+			<input class="input-candidate" type="text" name="Interviewer" autocomplete="on" placeholder="Interviewer" required />
+			<input class="input-candidate" type="date" name="Date" placeholder="dd/mm/yyyy" required/>
 		</form>
 
+		<div class="littleSpacing"></div>
```

Comment:
Never mind that silly thing, trying to modify all of the boxes in order to get rid of the spacing div